qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java

```diff
@@ -1214,7 +1215,7 @@ private void saveInfoDocument(String type, String value) throws IOException {
     private String loadInfoDocument(String type) throws IOException {
         Request request = new Request("GET", "/info/" + this.type + "/" + index + "_" + type);
         request.addParameter("filter_path", "_source");
-        if (isRunningAgainstAncientCluster()) {
+        if (getOldClusterVersion().before(Version.V_6_7_0)) {
```

Comment:
Whether a warning is expected, depends on whether type `doc` in `this.type` is used and that depends on whether the old cluster was before 6.7.0